### ISSUE TITLE
Can't use -mx11 on Fedora 34

### ISSUE THREAD

**Issue by billionai** [angband/angband#5088]
Not sure if this is the correct place to file this bug, but...

I'm trying to play angband on Fedora 34 (installed through RPMSphere) and the response is:
`angband: Couldn't load the requested font. (10x20)`

I have already tried installing xorg-x11-fonts-misc (which I think should be a dependency, if the default way to play requires it, but I digress), and tried locale-gen - which didn't work, but I don't know the correct command to get the same output.

Help please

**Comment by backwardsEric** [comment]
xorg-x11-fonts-misc looks like the correct package.  Does running

```
xlsfont -fn "10*"
```

from a terminal report anything on your system?  If xlsfont doesn't report that the font is available, angband won't find it.

I would have expected the rpm to handle the necessary steps to have the font server include the newly installed font.  Part 2.2 of [Xorg's font installation instructions](https://www.x.org/archive/X11R6.8.0/doc/fonts2.htmll) may help to get the font recognized by the font server.  This closed issue, https://github.com/angband/angband/issues/4575 , (about the 10x20 font not being available on Arch Linux) may also be useful.  On Arch, it seems it was also necessary to install xorg-mkfontscale.

**Comment by billionai** [comment]
`xlsfonts -fn "10*"` doesn't find the font indeed, so that is the issue

I tried using mkfontscale and mkfontdir, everything worked, but `xset fp+ <path>` gives me an error: does not exist, wrong permissions, no fonts.dir or incorrect font server/syntax; my guess is that it is this last point, but i don't know why or how to fix this

Extra information (not sure if it makes a difference) My system doesn't auto-start Xorg, so I ssh into it, `sudo startx`, then form another terminal I `ssh -X` into it. My best guess is that this is the culprit, but I can't really start the graphical interface with the system, because reasons. Anyway, this is clearly not an angband issue, so feel free to close it if you feel like it and thanks for the help. I'll look into this weird problem when I have more free time

**Comment by backwardsEric** [comment]
A quick search turned up this [>10 year old Fedora documentation](https://docs.fedoraproject.org/en-US/Fedora/12/html/Deployment_Guide/s2-x-fonts-core.html) for handling the core X11 fonts on Fedora.  I didn't spend much time to try to find anything newer.  Some of the other search results suggested that the font server, xfs, isn't installed by default on modern Fedora so that may be the issue.

From your description, the problem could also be a typo in the path you passed to xset.  On Debian bullseye it's in /usr/share/fonts/X11/misc, but that's likely different than where the rpm installed it.

**Comment by billionai** [comment]
I fixed it! This was an issue of my understanding of the Xorg server. The VM was not running any Xorg instances, and I thought it needed to run it to then forward the window to the host machine. I was wrong, when you use `ssh -X` from machine a to machine b, it uses the Xorg server of machine a. I had installed xorg-x11-fonts-misc on machine b. Installing it in machine a fixed it.

Thanks for the help!
